UnionPay/src/test/java/com/braintreepayments/api/UnionPayCardUnitTest.kt

```diff
@@ -244,17 +249,17 @@ class UnionPayCardUnitTest {
         sut.enrollmentId = "enrollmentId"
 
         val tokenizePayload = sut.buildJSON()
-        val creditCardPayload = tokenizePayload.getJSONObject("creditCard")
-        val optionsPayload = creditCardPayload.getJSONObject("options")
-        val unionPayEnrollmentPayload = optionsPayload.getJSONObject("unionPayEnrollment")
-
-        assertEquals("someCardNumber", creditCardPayload.getString("number"))
-        assertEquals("expirationMonth", creditCardPayload.getString("expirationMonth"))
-        assertEquals("expirationYear", creditCardPayload.getString("expirationYear"))
-        assertEquals("cvv", creditCardPayload.getString("cvv"))
-        assertFalse(optionsPayload.has("validate"))
-        assertEquals("enrollmentId", unionPayEnrollmentPayload.getString("id"))
-        assertFalse(unionPayEnrollmentPayload.has("smsCode"))
+        val creditCardPayload = tokenizePayload?.getJSONObject("creditCard")
```

Comment:
Looks like these tests are all just being updated to make these properties nullable. Curious why this is being done?

Reply:
Yeah good call, we may not need these in the PR. I'll double check.